data/api/client-server/rooms.yaml

```diff
@@ -105,9 +105,24 @@ paths:
           example: ""
           schema:
             type: string
+        - in: query
+          name: format
+          description: |-
+            The format to use for the returned data. `content` (the default) will
+            return only the content of the state event. `event` will return the entire
+            event in the usual format suitable for clients, including fields like event
+            ID, sender and timestamp.
+          example: event
+          schema:
+            type: string
+            enum:
+            - content
+            - event
       responses:
         "200":
-          description: The content of the state event.
+          description: |-
+            The content of the state event, or the entire client-formatted event
+            if `?format=event` was used.
           content:
             application/json:
               schema:
```

Comment:
this should have whatever the schema definition for `or` is

Reply:
I think I successfully told the schema to be an `oneOf`, but I'm not sure if the renderer actually supports that

Reply:
I don't think it did? https://pr2175--matrix-spec-previews.netlify.app/client-server-api/#get_matrixclientv3roomsroomidstateeventtypestatekey

Reply:
Yeah looks like it doesn't show the schema at all. Not sure if there's anything that can be done here. We don't show alternative schemas for other endpoints either, like /sync and /messages when using event_format=federation. Multiple examples would be nice, but I didn't see any way to do that either.

Reply:
yea... I think it's fine as a schema feature for now. We should file a bug that these `any|oneOf` things don't render though, if there isn't already one.